tsMuxer/metaDemuxer.cpp

```diff
@@ -287,13 +287,10 @@ int METADemuxer::addPGSubStream(const string& codec, const string& _codecStreamN
 std::vector<MPLSPlayItem> METADemuxer::mergePlayItems(const std::vector<MPLSParser>& mplsInfoList)
 {
     std::vector<MPLSPlayItem> result;
-    for (int i = 0; i < mplsInfoList.size(); ++i)
+    for (auto& i : mplsInfoList)
     {
-        const MPLSParser& mplsInfo = mplsInfoList[i];
-        for (int j = 0; j < mplsInfo.m_playItems.size(); ++j)
-        {
-            result.push_back(mplsInfo.m_playItems[j]);
-        }
+        const MPLSParser& mplsInfo = i;
+        for (auto& j : mplsInfo.m_playItems) result.push_back(j);
```

Comment:
Could be just `for (auto& j : i.m_playItems)`, it seems, no need for the additional variable.

Reply:
Done.